### PR TITLE
susefirewall2: don't generate multiline entries, it's against SuSEfir…

### PR DESCRIPTION
**Pull request by mgerstner**
susefirewall2: don't generate multiline entries, it's against SuSEfirrwall2 recommendation

multiline entries create trouble with fillup, see bsc#798468

see Configuration Hints in /etc/sysconfig/SuSEfirewall2

### CODE REVIEW

File: library/network/src/lib/network/susefirewall2.rb [hunk 1]
```diff
@@ -261,14 +261,6 @@ def main
 
     # <!-- SuSEFirewall GLOBAL FUNCTIONS USED BY LOCAL ONES //-->
 
-    # Returns whether records in variable should be written one record on one line.
-    # @return [Boolean] if wolpr
-    def WriteOneRecordPerLine(key_name)
-      return true if key_name.nil? && key_name == ""
-
-      Builtins.contains(@one_line_per_record, key_name)
```
Comment (lslezak) [inline library/network/src/lib/network/susefirewall2.rb]: It seems that the `@one_line_per_record` value is not used any more. Please remove it as well.

**Comment by lslezak** [comment]
Please also add a `changes` file entry and raise the package version (run `rake version:bump`).

File: library/network/src/lib/network/susefirewall2.rb (hunk 1 above)
Comment (mgerstner) [inline library/network/src/lib/network/susefirewall2.rb]: Okay I've removed it

**Comment by mgerstner** [comment]
Okay, hope I got that right after some woes with conflicts and sync merging

**Comment by lslezak** [comment]
Thanks for your contribution! :+1:

**Pull request merged**